<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>change_lang</class>
 <widget class="QWidget" name="change_lang">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>454</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>775</width>
    <height>454</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>775</width>
    <height>454</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>choice language</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/rec/pictures/free-icon-atm-5346812.png</normaloff>:/rec/pictures/free-icon-atm-5346812.png</iconset>
  </property>
  <widget class="QGroupBox" name="wait_group">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>-20</y>
     <width>431</width>
     <height>491</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>454</height>
    </size>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string>GroupBox</string>
   </property>
   <widget class="QGraphicsView" name="graphicsView_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>411</width>
      <height>391</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background: transparent</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>440</y>
      <width>151</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>Wait...</string>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>70</y>
     <width>771</width>
     <height>161</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">border:none;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="flat">
    <bool>false</bool>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>761</width>
      <height>73</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="russianBtn">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>71</height>
        </size>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: qlineargradient(spread:pad, x1:0.122749, y1:1, x2:0.129, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #006400;
font-size: 22px;
font-weight: 700;
margin-left:15px;
margin-right: 10px;
border-radius: 10px;</string>
       </property>
       <property name="text">
        <string>Русский</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="englishBtn">
       <property name="minimumSize">
        <size>
         <width>221</width>
         <height>71</height>
        </size>
       </property>
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="autoFillBackground">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color:qlineargradient(spread:pad, x1:0.921631, y1:1, x2:0.917, y2:1, stop:0 rgba(0, 91, 0, 255), stop:1 rgba(211, 211, 211, 255));
color: #006400;
font-size: 22px;
font-weight: 700;
margin-left:10px;
margin-right: 15px;
border-radius: 10px;


</string>
       </property>
       <property name="text">
        <string>English</string>
       </property>
       <property name="icon">
        <iconset resource="res.qrc">
         <normaloff>:/rec/pictures/free-icon-play-4210629.png</normaloff>:/rec/pictures/free-icon-play-4210629.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="default">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>-11</x>
     <y>0</y>
     <width>791</width>
     <height>91</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:white;
background-color:#006400;</string>
   </property>
   <property name="text">
    <string>Выберите язык 
Choose language</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections/>
</ui>
